examples/bzlmod/requirements.in

```diff
@@ -1,5 +1,5 @@
 requests~=2.25.1
 s3cmd~=2.1.0
-yamllint~=1.26.3
+yamllint>=1.28.0
```

Comment:
was this required because the old yamllint package doesn't work on debian11 ?

Reply:
It was failing to build from source on Debian. I assume it's because of the non-hermeticity of building wheels on repository rules. 1.28 release has a pre-built wheel.